tests/unit_tests/scenarios/test_summary_response.py

```diff
@@ -67,7 +62,10 @@ def setup_configuration(tmpdir):
         run_sim(ref_time, 0.5)
         ert_config = ErtConfig.from_file("config.ert")
         ert = EnKFMain(ert_config)
-        yield ert
+        ensemble = storage.create_experiment(
+            parameters=ert_config.ensemble_config.parameter_configuration
+        ).create_ensemble(ensemble_size=100, name="prior")
+        yield ert, ensemble
```

Comment:
Stylistic nitpick, but you can keep `prior_ensemble` as a separate fixture by having it depend on `setup_configuration`. This fixture will be run exactly once per test, so the tests that depend on both `setup_configuration` and `prior_ensemble` will get the same exact ERT object as `prior_ensemble` did.